src/machine/machine_atsamd21.go

```diff
@@ -102,11 +118,15 @@ type UART struct {
 }
 
 var (
+	// UART0 is actually a USB CDC interface.
+	//UART0 = UART{Buffer: NewRingBuffer(), UsesUSB: true}
```

Comment:
Is this comment here intentional, with `UsesUSB`?

Reply:
No that was a comment that should have been removed.